tests/unit/commands/test_runners.py

```diff
@@ -43,10 +43,25 @@ def test_parse_all(self, cmdline_test):
             with pytest.raises(cmdexc.NoSuchCommandError):
                 list(cr.parse_all(cmdline_test.cmd, aliases=False))
 
+    def test_parse_all_with_alias(self, cmdline_test, monkeypatch):
+        alias_dict = dict(alias_name=cmdline_test.cmd)
+
+        def mock_get(section, *args, **kwargs):
+            assert section == "aliases"
+            return alias_dict.get(*args, **kwargs)
+        monkeypatch.setattr("qutebrowser.config.config.get", mock_get)
```

Comment:
I'm not sure if I ever used it for a list in the config (like the `aliases` section is), but I think you can simplify this by getting a `config_stub` [fixture](http://pytest.org/latest/fixture.html) and simply doing `config_stub.data = {'aliases': {'alias_name': cmdline_test.cmd}}`.